src/components/navigation/test/Pagination-test.js

```diff
@@ -0,0 +1,132 @@
+import { checkAccessibility, mount } from '@hypothesis/frontend-testing';
```

Comment:
This is unchanged from `PaginationNavigation` in the client aside from removing a few blank lines and unnecessary `act` calls.